wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -97,49 +200,58 @@ int tsip_hw_lock()
 */
 void tsip_hw_unlock( void )
 {
-    WOLFSSL_MSG("enter tsip_hw_unlock");
-    /* unlock hw engine for next use */
+    /* WOLFSSL_MSG("enter tsip_hw_unlock"); */
```

Comment:
Please either remove or uncomment this debug line, and the corresponding one below.

Reply:
They will be removed.